tests_config/run_spec.js

```diff
@@ -111,140 +111,158 @@ global.run_spec = (fixtures, parsers, options) => {
 
   const stringifiedOptions = stringifyOptions(options);
 
```

Comment:
This file's diff will look better if you [hide workspace changes](https://github.com/prettier/prettier/pull/8408/files?diff=unified&w=1).